mlflow/pipelines/steps/predict.py

```diff
@@ -133,6 +133,19 @@ def from_pipeline_config(cls, pipeline_config, pipeline_root):
                 "Invalid `output_format` in predict step configuration.",
                 error_code=INVALID_PARAMETER_VALUE,
             )
+        if "model_uri" not in step_config:
```

Comment:
At L131, what does `table` mean? I don't see it's actually supported anywhere.

Reply:
It is being used https://github.com/mlflow/mlflow/blob/0321d828af8f908c5ba7141353acd25cf8bcb114/mlflow/pipelines/steps/predict.py#L105-L106 to save the scored dataframe as a delta table.